doc/source/templates.rst

```diff
@@ -42,12 +42,10 @@ Done!
 
 **Customizing a template:**
 
-To make further customizations or just patch the template, there are two
-options:
+To make further customizations or just patch the template, you'll need
+to convert the template back into a jail, start it, and then convert it
+back to a template once you've completed your changes.
```

Comment:
n.b.: This only works so long as no jail has been created from this template.

Reply:
That's a great point. I'll rework this text, probably to indicate that if jails have already been created from the template then the best course of action would be clone the template into a new template, make the changes, and then create new jails from that.

Reply:
I recommend using a thickjail for that clone with the new template. Otherwise it too will be dependent on the old living.